sdk/src/commitment_config.rs

```diff
@@ -46,11 +46,28 @@ impl CommitmentConfig {
 
 #[derive(Serialize, Deserialize, Clone, Copy, Debug, PartialEq, Eq, Hash)]
 #[serde(rename_all = "camelCase")]
+/// An attribute of a slot. It describes how finalized a block is at some point in time. For example, a slot
```

Comment:
@CriesofCarrots @jstarry, how's this look?  I don't know if we've ever stated "commitment level means ...".  I found it quite difficult to describe - maybe a red flag that I got it wrong.